tests/scripts/all.sh

```diff
@@ -1608,6 +1608,15 @@ component_test_memsan_constant_flow () {
     make test
 }
 
+skip_non_constant_flow_components () {
+    # The following components are especially slow and don't have any constant-flow annotations,
+    # so disable them for better CI performance.
+    # It would be better to pass a list of tests that we want to run, rather than tests to skip,
+    # but this functionality is currently not implemented.
+    SKIP_TEST_SUITES='ccm,dhm,ecdh,ecdsa,ecjpake,ecp,lms,gcm,ssl,pk,pkcs1_v21,pkcs5,pkparse,psa_crypto,psa_crypto_storage_format,rsa,x509parse,x509write'
+    export SKIP_TEST_SUITES
```

Comment:
We could discover the test suites with constant-flow tests automatically. This would be more robust than an ad hoc list where selection seems haphazard.
```
# Skip the test suites that don't have any constant-flow annotations.
SKIP_TEST_SUITES=$(
    grep -L TEST_CF_ tests/suites/test_suite_*.function |
    sed 's!.*/test_suite_!!; s!\.function$!!' |
    tr '\n' ,)
export SKIP_TEST_SUITES
```

Reply:
Agreed, I thought about doing this as a second step. Thanks.